src/helpers/routes/routes.js

```diff
@@ -13,6 +13,15 @@ class APIRoutes {
   static veteransSignUpPath() {
     return APIRoutes.create('/veterans');
   }
+  static veteransConnectSignUpPath(id) {
+    return APIRoutes.create(`/veterans/${id}/connect_sign_up`);
+  }
+  static veteransConnectStatus(id) {
+    return APIRoutes.create(`/veterans/${id}`);
+  }
+  static veteransMilitaryBranch() {
+    return APIRoutes.create('/veterans/get_military_branch');
```

Comment:
do you have a PR where you define the military_branch enum in the veteran model?

Reply:
and where is the endpoint on the rails end?

Reply:
o oops forgot to create PR for web, done now